scripts/map.js

```diff
@@ -20,13 +20,20 @@ var zAcres = 'MA' // municipal area
 
 var style = function (filters, feature) {
   var opacity = $('input[name="opacity"]').val() / 100
-
+  let fillColor = satisfiesFilters(filters, feature)
+  ? zone2color[feature.properties[zType]]
+  : zone2color['NS'];
+
+  // If the feature is "Not Zoned" the properties[zType] will be null
+  // This fixes the null areas being blue by default
+  if(feature.properties[zType] === null)
```

Comment:
[Suggestion] can we instead modify satisfiesFilters() to handle the null case? This might clean up some code here and we won't have to hardcode fillColor here, we'd be able to use the zone2color map above instead.

Reply:
I'm not sure how satisfiesFilters returning true for the null case would affect the following:
```
dataLayer.eachLayer(function (l) {
    if (l.feature.properties[zTown] === townActive) {
      totalAcres += l.feature.properties[zAcres] || 0
      if (satisfiesFilters(filters, l.feature)) {
        satisfiesAcres += l.feature.properties[zAcres] || 0
      }
    }
  })
```

Reply:
I moved the hardcoded value into zone2color so they're at least all in one spot. Let me know what you think